examples/example.c

```diff
@@ -106,6 +112,14 @@ main(int argc, char **argv)
         }
     }
 
+    if (has_arg(argc, argv, "sleep")) {
+#ifdef SENTRY_PLATFORM_WINDOWS
+        Sleep(10 * 1000);
+#else
+        sleep(10);
+#endif
```

Comment:
Maybe move these platform dependent defines to a separate file/header that contains e.g.:
```
#ifdef SENTRY_PLATFORM_WINDOWS
        #define sentry_sleep_s(SECONDS) Sleep((SECONDS) * 1000)
#else
        #define sentry_sleep_s(SECONDS) sleep(SECONDS)
#endif
```

Reply:
hm, I don’t really want to expose/maintain these things as a *public API*.

Reply:
but I moved it to the top of the file, so the platform check is not duplicated ;-)

Reply:
I didn't see this was code from the example.
Good solution!